src/OrchardCore.Modules/OrchardCore.Queries/Sql/ViewModels/AdminQueryViewModel.cs

```diff
@@ -6,6 +6,8 @@ namespace OrchardCore.Queries.Sql.ViewModels
 {
     public class AdminQueryViewModel
     {
+        public static string MatchAllQueryBase64 { get; } = Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(@"SELECT * FROM ContentItemIndex"));
```

Comment:
```suggestion
        public static string MatchAllQueryBase64 => Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(@"SELECT * FROM ContentItemIndex"));
```

Reply:
That would evaluate on each call. Which was done before this change.